nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/components/nxql-tab/nxql-tab.component.spec.ts

```diff
@@ -344,4 +345,13 @@ describe("NXQLTabComponent", () => {
       expect(result.labels.submitBtnLabel).toBe(PICTURE_RENDITIONS_LABELS.RENDITIONS_BUTTON_LABEL);
     });
   });
+
+  describe('FEATURES.THUMBNAIL_GENERATION', () => {
+    it('should return correct labels and data for NXQL tabType', () => {
+      const result = featureMap()[FEATURES.THUMBNAIL_GENERATION](GENERIC_LABELS.NXQL);
+      expect(result.labels.pageTitle).toBe(THUMBNAIL_GENERATION_LABELS.NXQL_THUMBNAIL_GENERATION_TITLE);
+      expect(result.labels.submitBtnLabel).toBe(THUMBNAIL_GENERATION_LABELS.THUMBNAIL_GENERATION_BUTTON_LABEL);
+      expect(result.data.bodyParam.query).toBe(THUMBNAIL_GENERATION_LABELS.NXQL_QUERY);
+    });
+  });
```

Comment:
why only one test case per file ? There should be more

Reply:
will be providing this separate PR.